Releasing.md

```diff
@@ -0,0 +1,30 @@
+Release process.
+
+Only for Maintainers.
+
+1. Tag with version to be released eg: git tag -a 0.4.0-beta -m ".4.0.0-beta"
```

Comment:
".4.0.0-beta" -> "0.4.0-beta"

Reply:
and, i would try to add/test the `--sign` or `-s` to generate a verified tag. (that doesn't change a thing AFAIK)

Reply:
```suggestion
1. Tag with version to be released eg: git tag -a 0.4.0-beta -m "0.4.0-beta"
```

Reply:
updated!